v3/cmd/zlint/main.go

```diff
@@ -192,10 +207,17 @@ func trimmedList(raw string) []string {
 // use.
 func setLints() (lint.Registry, error) {
 	// If there's no filter options set, use the global registry as-is
-	if nameFilter == "" && includeNames == "" && excludeNames == "" && includeSources == "" && excludeSources == "" {
```

Comment:
Just a style change because this boolean condition was getting long.

Reply:
Nice, looks good :-)